src/viam/examples/modules/complex/main.cpp

```diff
@@ -29,32 +29,28 @@ using namespace viam::sdk;
 
 int main(int argc, char** argv) {
     API base_api = API::get<Base>();
+    API gizmo_api = API::get<Gizmo>();
+    API summation_api = API::get<Summation>();
     Model mybase_model("viam", "base", "mybase");
 
+    // Make sure to explicity register resources with custom APIs.
+    Registry::register_resource<GizmoClient, GizmoServer, GizmoService>(gizmo_api);
+    Registry::register_resource<SummationClient, SummationServer, SummationService>(summation_api);
```

Comment:
I find it a little odd that in `complex/client`, the registration must make reference to `[Gizmo,Summation]Server` despite not really needing anything to do with servers, and that in the module main that the registration must make reference to `[Gizmo,Summation]Client`, which it will presumably never use.

Reply:
Hmm yeah I agree. Will revisit and see if we can clean that up some.

Reply:
It makes me wonder whether client and server registration should be separate activities.